app/views/components/docs/attachment_meta.yml

```diff
@@ -0,0 +1,33 @@
+name: Attachment meta
+description: Displays an attachment with the associated metadata and possible actions
+body: |
+  Metadata contains title, file format, file size, number of pages. Actions can be insert, preview, edit and delete.
+  The component is dependent on the govuk-frontend grid system.
+accessibility_criteria: |
+  The image must:
+
+  * have alt text
+examples:
+  default:
+    data:
+      url: https://www.gov.uk/government/publications/minimum-wage-jobs-in-ashfield-local-authority-since-2009
+      thumbnail_src: https://assets.digital.cabinet-office.gov.uk/government/assets/pub-cover-doc-afe3b0a8a9511beeca890340170aee8b5649413f948e512c9b8ce432d8513d32.png
+      title: Document attachment
+      metadata_items:
+       - file_format: MS Word Document
```

Comment:
We're going to hit something tricky with this and govspeak.

At the moment the information on the various texts to display here is stored in govspeak: https://github.com/alphagov/govspeak/blob/master/lib/govspeak/presenters/attachment_presenter.rb#L203-L240 (presumably Whitehall has a similar thing).

I don't really know what the right answer is for where to put this is.

This is a bit outside the scope of your card, just mentioning it as I see a problem looming which may make it harder to use this component 🤔

Reply:
Could we render the attachment using Govspeak and append the actions separately?